stdlib/public/Cxx/CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ add_swift_target_library(swiftCxx STATIC NO_LINK_NAME IS_STDLIB IS_SWIFT_ONLY
     # For functionality that depends on the C++ stdlib, use C++ stdlib overlay (`swiftstd` module).
     -Xcc -nostdinc++
 
+    # The varying modularization of the C++ standard library on different
+    # platforms makes it difficult to enable MemberImportVisibility for this
+    # module
+    -disable-upcoming-feature MemberImportVisibility
```

Comment:
I think this flag should probably only be on the CxxStdlib module. The Cxx module doesn't import the C++ standard library and is mostly platform-independent.